src/plugin/regions/region.js

```diff
@@ -203,8 +203,7 @@ export class Region {
                 cursor: 'col-resize',
                 position: 'absolute',
                 top: '0px',
-                width: '1%',
-                maxWidth: '4px',
+                width: '2px',
```

Comment:
I don't know if a fixed width would be ideal for all people, because it can be for example too large for someone.. But I'm agree with you that with a width of 1%, at some point the handle disappeared.. But maybe it also can be passed from outside, so everyone can define a custom width?

Reply:
These are default CSS styles that get applied if you don't provide any, so yes you're right you can provide a custom width. I can change it to width: 1%, minWidth: 1px if you think most people would want the handle size to scale with the region

Reply:
I talk about of another prop, which you can pass from outside, so you can customize not only the defined regions but also those you draw dragging. But maybe it is outside the purposes of this PR

Reply:
Sounds like a good idea. The region dragging can be improved a lot, we can make another PR later to fix that.